server/insertEvent.js

```diff
@@ -1,8 +1,7 @@
 // Insert Event function connects to Database //
-
+require('dotenv').config()
```

Comment:
You can just include this `require('dotenv').config()` once, in your `index.js`.